src/ProcessManager.php

```diff
@@ -222,6 +222,13 @@ class ProcessManager
      */
     const CONTROLLER_PORT = 5500;
 
+    /**
+     * TCP context options.
+     *
+     * @var array
+     */
+    protected $tcp_context = [];
```

Comment:
Could you make this `$tcpContext` in line with the other options?

Reply:
I updated var names !